### PR TITLE
[IMP] mrp: assign lot/serial number to manuf products

### PR DESCRIPTION
**Pull request by Felicious**
### Summary of changes
- Add section to 17.0 [Configure manufacturing product](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.html) doc to include how to set up manufacturing products to be tracked by lots and serial numbers
- Replace `ref` links to BoM doc with a relative path link using the `doc` directive
- Merge after https://github.com/odoo/documentation/pull/9239

[Task](https://www.odoo.com/web#id=3854466&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

**Note for reviewers**: does the structure of the info make sense? Do you think there's a better place to include this info? The lots/serial numbers config section is optional, but does it come off that way in the doc? If not, where in the manufacturing docs should I put this info? In my opinion, he section is too short to be its own doc, but too long to put in a tip admonition block. Let me know what you think!

### CODE REVIEW

**Comment by Felicious** [comment]
Hello @ksc-odoo ! This PR is ready for your first round of peer review (: I have a note for you in the PR description -- appreciate your input in structuring the info! :blush:

**Review by ksc-odoo — APPROVED** [review]
Great work, @Felicious -- everything seems good to me. And the current placement of this information seems fine to me -- but it also, in my opinion, does **not** seem too much for an admonition block, if you think it would be better placed elsewhere. Either way, this PR was very direct, too the point, and void of any suggestions (especially since it was so brief 😄 ). Once you consider my comments about the potential placement (which I hope are helpful), feel free to tag this for Tech Review whenever you're ready. Thanks!

File: content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst [hunk 1]
```diff
@@ -24,6 +24,35 @@ checkbox in the :guilabel:`Routes` section. This tells Odoo the product can be m
    :align: center
    :alt: The Manufacturing route on the Inventory tab of a product page.
 
+.. _manufacturing/basic_setup/lot-serial-tracking:
+
+Lot/serial number tracking
+--------------------------
+
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst:31]: The assignment of lots or serial numbers to newly manufactured products is **optional**.

Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst:31]: That could be used as a lead-in sentence to this section to *really drive home* that point -- but that's completely up to you

**Review by ksc-odoo — APPROVED** [review]

File: content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst (hunk 1 above)
Comment (Felicious) [reply content/applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst:31]: This is a great suggestion! Will go with this

**Comment by Felicious** [comment]
Hi @samueljlieber ! This is a tiny update to the `configure_manufacturing_product` file that's intended to merge after the BoM doc (:

**Review by samueljlieber — APPROVED** [review]
Hi @Felicious, I think the information is presented in a good way, as is. It is clear to me that this step is an optional configuration 👍. 

I have a suggestion about the images added in this PR– IMO they are not needed for this section, and including them breaks up the flow of the instruction too much. Its up to you to keep or remove them, but I think your written instructions are very clear and provide enough coverage :) WDYT?

I am approving and delegating merge to you, so that you can once https://github.com/odoo/documentation/pull/9239 is merged. 

Thanks for your hard work!
..
@robodoo delegate=Felicious

**Comment by samueljlieber** [comment]
p.s. per robot's comment please re-target the PR to 17.0 once ready before merge- tag me again so I can delegate :)

**Comment by samueljlieber** [comment]
Thanks @Felicious, looks good 👍
@robodoo delegate=Felicious

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**